etc/chasquid/hooks/post-data

```diff
@@ -86,10 +86,12 @@ if [ "$AUTH_AS" != "" ] && command -v dkimsign >/dev/null; then
 	DOMAIN=$( echo "$MAIL_FROM" | cut -d '@' -f 2 )
 	if [ -f "domains/$DOMAIN/dkim_selector" ] \
 			&& [ -f "certs/$DOMAIN/dkim_privkey.pem" ]; then
-		dkimsign -n -hd \
-			-key "certs/$DOMAIN/dkim_privkey.pem" \
-			-s "$(cat "domains/$DOMAIN/dkim_selector")" \
-			-d "$DOMAIN" \
-			< "$TF"
+		dkimsign \
+			"$(cat "domains/$DOMAIN/dkim_selector")" \
+			"$DOMAIN" \
+			"certs/$DOMAIN/dkim_privkey.pem" \
+			< "$TF" > "${TF}.signed"
+		echo "$( comm -13 "$TF" "${TF}.signed" )"
```

Comment:
I am working on an equivalent patch that supports both dkimpy and driusan/dkim, and ran into an issue here.

Specifically, this doesn't seem to work reliably, and doesn't always output the new header (sometimes, it shows more things, which can be problematic).

For now the best I have (which is awful) is to do:

```
diff "$TF" "$TF.dkimout" | grep '^> ' | sed 's/^> //g'
```

I'm mentioning in case you're using this hook in your setup, it's very possible that it's doing the wrong thing.